Rockerfile

```diff
@@ -47,12 +45,12 @@ ENV PATH=/opt/rocker/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin
 
 CMD ["/opt/rocker/bin/rocker"]
 
-PUSH dockerhub.grammarly.io/rocker:{{ .Version }}
+{{ if $version }}
```

Comment:
I think we should do a branch `PUSH` always, without any conditions. Branch can be `none` or `no-branch` by default.
PUSH with version can be with condition and be pushed to `rocker:$version` but not `rocker:$branch-$version`. Because if you know a particular version, you don't care about a branch.